parsec/core/cli/pki.py

```diff
@@ -105,6 +106,10 @@ async def _pki_enrollment_poll(
     save_device_with_selected_auth: Callable,
     extra_trust_roots: Sequence[Path],
 ):
+
+    # Use extra trust roots from both command line and config
+    extra_trust_roots = [*extra_trust_roots, *config.pki_extra_trust_roots]
```

Comment:
Why not add this directly in the `@core_config_options` decorator ?

BTW I think it would be better to have the option name `--extra-trust-root` mention it is related to PKI, so for instance `--pki-extra-trust-root` ? (and adding a `help="Additional directory to look for PKI root certificate"`)